.gitmodules

```diff
@@ -13,3 +13,7 @@
 [submodule "packages/charls/extern/charls"]
 	path = packages/charls/extern/charls
 	url = https://github.com/cornerstonejs/charls.git
+[submodule "packages/libjxl/extern/libjxl"]
```

Comment:
Uses the v0.11.x release tag so I don't think we need a custom version tag here.